Can't resolve 'child_process' in tiny-worker/lib

Hi and thanks for your work. The library saves a lot of effort on building isomorphic apps

I have an issue completely similar to the one described in #183. Even the configs are very similar.
I've completed all the steps but after adding all the modules to the IgnorePlugin I still get errors.
Webpack considers node packages missing.
```
Error: Cannot find module 'os'
    at webpackMissingModule (webpack://%5Bname%5D.js/./node_modules/threads/dist-esm/master/implementation.node.js?:9:45)
     at eval (webpack://%5Bname%5D.js/./node_modules/threads/dist-esm/master/implementation.node.js?:9:122)
     at Module../node_modules/threads/dist-esm/master/implementation.node.js
     at __webpack_require__ 
     at eval (webpack://%5Bname%5D.js/./node_modules/threads/dist-esm/master/implementation.js?:6:78)
     at Module../node_modules/threads/dist-esm/master/implementation.js
     at __webpack_require__
     at eval (webpack://%5Bname%5D.js/./node_modules/threads/dist-esm/master/index.js?:4:73)
```
I use target: "node". Is there any info I can provide to resolve this?

--- a/src/loader.js
+++ b/src/loader.js
@@ -59,7 +59,9 @@ export function pitch (request) {
   const workerCompiler = this._compilation.createChildCompiler(NAME, workerOptions, plugins);
   workerCompiler.context = this._compiler.context;
 
-  if (pluginOptions.target && /\bnode\b/.test(pluginOptions.target)) {
+  const target = pluginOptions.target || compilerOptions.target
+
+  if (target && /\bnode\b/.test(target)) {
     new NodeTargetPlugin().apply(workerCompiler);
   }
   (new WebWorkerTemplatePlugin(workerOptions)).apply(workerCompiler);

